client/src/views/client-area/DashboardPage.vue

```diff
@@ -1,18 +1,520 @@
+                  />
+                  <ion-skeleton-text
+                    :animated="true"
+                    class="invoices-organization"
+                  />
+                  <ion-skeleton-text
+                    :animated="true"
+                    class="invoices-amount"
+                  />
+                  <ion-skeleton-text
+                    :animated="true"
+                    class="invoices-status"
+                  />
+                </div>
+              </ion-item>
+            </ion-list>
+          </div>
+        </div>
+        <ion-text
+          class="body-lg"
+          v-else
+        >
+          {{ $msTranslate('clientArea.dashboard.invoices.noInvoices') }}
+        </ion-text>
+      </div>
+    </div>
+
+    <!-- payment section -->
+    <div class="dashboard-section payment-container">
+      <div class="dashboard-section-title">
+        <ion-title class="title-h2">{{ $msTranslate('clientArea.dashboard.payment.title') }}</ion-title>
+        <ion-text class="payment-text body">
+          {{ $msTranslate('clientArea.dashboard.payment.description') }}
+        </ion-text>
+      </div>
+      <div class="payment">
+        <ms-stripe-card-details
+          v-if="defaultCard"
+          :card="defaultCard"
+          class="payment-card"
+        />
+        <div
+          class="skeleton-loading"
+          v-else
+        >
+          <ion-skeleton-text
+            :animated="true"
+            class="skeleton-loading-card"
+          />
+        </div>
+        <ion-text
+          class="custom-button custom-button-fill button-medium"
+          @click="switchPage(ClientAreaPages.PaymentMethods)"
+        >
+          {{ $msTranslate('clientArea.dashboard.payment.update') }}
+        </ion-text>
+      </div>
+    </div>
   </div>
 </template>
 
 <script setup lang="ts">
-import { BmsOrganization } from '@/services/bms';
+import { IonTitle, IonText, IonList, IonItem, IonIcon, IonSkeletonText } from '@ionic/vue';
+import { download } from 'ionicons/icons';
+import { ClientAreaPages } from '@/views/client-area/types';
+import {
+  BmsAccessInstance,
+  DataType,
+  BmsOrganization,
+  OrganizationStatsResultData,
+  BmsInvoice,
+  BillingDetailsPaymentMethodCard,
+  PaymentMethod,
+} from '@/services/bms';
+import { MsInformationTooltip, I18n, MsStripeCardDetails } from 'megashark-lib';
+import { onMounted, ref } from 'vue';
+import { PaymentMethod as MsPaymentMethod } from 'megashark-lib';
+import { navigateTo, Routes } from '@/router';
+import { DateTime } from 'luxon';
 
-defineProps<{
+const props = defineProps<{
   organization: BmsOrganization;
 }>();
+
+const stats = ref<OrganizationStatsResultData | undefined>(undefined);
+const invoices = ref<Array<BmsInvoice>>([]);
+const defaultCard = ref<MsPaymentMethod.Card | undefined>(undefined);
+const currentDate = DateTime.now();
+
+// should be fetched from the backend
+const ADMIN_USER_PRICE = 15;
```

Comment:
Since this is only used in a mock function, remove it and put the value directly in the function. This will make it easier to remove the function later on, don't have to worry about wandering variables.